packages/core/android/build.gradle

```diff
@@ -169,10 +169,10 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   compileOnly "com.squareup.okhttp3:okhttp:3.12.13"
   
-  implementation "com.datadoghq:dd-sdk-android-rum:2.2.0"
-  implementation "com.datadoghq:dd-sdk-android-logs:2.2.0"
-  implementation "com.datadoghq:dd-sdk-android-trace:2.2.0"
-  implementation "com.datadoghq:dd-sdk-android-webview:2.2.0"
```

Comment:
Can you make this change part of another commit? that will make it easier to revert if it needs to be reverted.

Reply:
done 👍